features/ubuntu_pro_fips.feature

```diff
@@ -0,0 +1,150 @@
+Feature: Command behaviour when auto-attached in an ubuntu PRO fips image
+
+    @series.lts
+    @uses.config.machine_type.aws.pro.fips
```

Comment:
@orndorffgrant @renanrodrigo do you think we need the proxy test for the fips pro machines ? I think it is not necessary, but I want to hear the team's opinion on that

Reply:
I agree. The only difference is the crypto available, but that is unlikely to interfere with proxy usage. Or at least, if there is a crypto problem, then my guess is that it would probably cause other bigger issues first.

Reply:
@CalvoM let's remove that test then